doc/introduction/chemistry.rst

```diff
@@ -1,10 +1,10 @@
 Quantum Chemistry
 =================
 
-PennyLane provides a quantum chemistry module ``qchem`` to perform quantum simulations
-of the electronic structure of molecules. ``qchem`` contains tools to construct the
-electronic Hamiltonian of molecules, and uses PennyLane to implement the Variational Quantum
-Eigensolver (VQE) algorithm.
+PennyLane provides a differentiable Hartree-Fock solver module :mod:`~.hf` and a quantum chemistry
```

Comment:
We should eventually do a larger revamp of this page to better reflect the overall current functionality, but this is better left for another PR

Reply:
yep agree!